subworkflows/local/metagenomics.nf

```diff
@@ -0,0 +1,62 @@
+include { METAGENOMICS_COMPLEXITYFILTER } from './metagenomics_complexityfilter'
+include { METAGENOMICS_PROFILING        } from './metagenomics_profiling'
+include { METAGENOMICS_POSTPROCESSING   } from './metagenomics_postprocessing'
+
+workflow METAGENOMICS {
+    take: ch_bamfiltered_for_metagenomics
+
+    main:
+    // Define channels
+    ch_multiqc_files                = Channel.empty()
+    ch_versions                     = Channel.empty()
+
+    // Add single_end parameter to meta.
+    // Reads were merged before, so single_end is always true!
```

Comment:
It would be good to double check this is always correct... we allow e.g. paired-end mapping, and tools like Kraken allow paired-end classification

Reply:
checked. (see most recent PR comment) - it is always correct, based on the current implementation of bamfiltering --> metagenomics

However, this functionality can be changed, but would be better to add it as a extension after metagenomics is merged